br2-external/configs/zcu106_defconfig

```diff
@@ -50,7 +50,7 @@ BR2_TOOLCHAIN_BUILDROOT_CXX=y
 BR2_TOOLCHAIN_BUILDROOT_FORTRAN=y
 BR2_GCC_ENABLE_LTO=y
 BR2_ROOTFS_DEVICE_CREATION_DYNAMIC_MDEV=y
-BR2_ROOTFS_OVERLAY="${BASE_DIR}/board/alkali/overlay/"
+BR2_ROOTFS_OVERLAY="${BASE_DIR}/overlay/ ${BR2_EXTERNAL_NVME_FIRMWARE_PATH}/board/zcu106/overlay"
```

Comment:
Is the whitespace in the quoted path intentional ?

Reply:
Yes. Those two paths should be separated with whitespace